src/scheduler/resource.cpp

```diff
@@ -96,129 +88,72 @@
 
 /**
  * @brief
- * 		query a pbs server for the resources it knows about
+ * 		query a pbs server for the resources it knows about and fill in the global unordered_map
  *
  * @param[in]	pbs_sd	-	communication descriptor to pbs server
  *
- * @return	resdef**
- * @retval	array of resources from server
- * @retval	NULL	: on error
+ * @return	bool
```

Comment:
Looks like you are returning an unordered map from this function.

Reply:
ahh yeah.  When I originally modified this function, I had it directly modify allres.  I then realized I needed the old and new maps so I could update the defs in last_running.  I'll update it.